src/components/common/constants.ts

```diff
@@ -1,37 +1,52 @@
 export const NETWORK_LIST = [
     {
-      name: "Mainnet",
```

Comment:
These changes are due to inconsistent prettier settings. We should create a prettierrc.json file in another PR

Reply:
Noted, will add a `prettierrc.json` in the coming days.